brokenspoke_analyzer/core/ingestor.py

```diff
@@ -123,18 +123,14 @@ def load_water_blocks(engine: Engine, csvfile: pathlib.Path) -> None:
 
 def delete_water_blocks(engine: Engine) -> None:
     """Delete the water blocks located within the city boundaries."""
-    query = (
-        f"DELETE FROM {CENSUS_BLOCKS_TABLE} AS blocks "
-        f"USING {WATER_BLOCKS_TABLE} AS water "
-        "WHERE blocks.BLOCKID10 = water.geoid;"
-    )
+    query = f"DELETE FROM {CENSUS_BLOCKS_TABLE} AS blocks WHERE blocks.aland20 = 0;"
```

Comment:
I like that very much! This would allow us to stop generating the waterblocks manually. We are going to double check that it produces identical or similar results, but otherwise, I like it.

Reply:
I forgot to double check this, but it appears at least for Wisconsin, there are a very small percentage of (24 out of 9,680) blocks where `aland20` is 0, but the population is greater than zero. Would it be preferable to keep the populated blocks?

Reply:
We'll check how we generate them, and we'll get back to you. We could then compare the two methods.

Reply:
Alright, so we do the exact same thing, but for some historical reasons we do it in a separate process... 🤔 Therefore we can totally keep what you did, it will consolidate and simplify the process.

We don't need to account for the population when `aland20` is 0.

Reply:
Sounds good, will leave it as is!

Reply:
To cleanup a bit more, could you remove the `def load_water_blocks()` and its references, as well as the `WATER_BLOCKS_TABLE` constant please?

Reply:
Yep, I think I got all of them